Singularity/Singularity/Units/MilitaryUnit.cs

```diff
@@ -1,4 +1,6 @@
 ﻿using System;
```

Comment:
i think they'll be quite a few merge conflicts in the class

Reply:
None that i see